examples/windows_launch_templates/main.tf

```diff
@@ -0,0 +1,114 @@
+provider "aws" {
+  region = local.region
+}
+
+locals {
+  name            = "launch_template-${random_string.suffix.result}"
+  cluster_version = "1.20"
+  region          = "eu-west-1"
+}
+
+################################################################################
+# EKS Module
+################################################################################
+
+module "eks" {
+  source                          = "../.."
+  cluster_name                    = local.name
+  cluster_version                 = local.cluster_version
+  subnets                         = module.vpc.private_subnets
+  vpc_id                          = module.vpc.vpc_id 
+
+  worker_groups_launch_template = [
+    {
+      name                          = "windows"
+      platform                      = "windows"
+      instance_type                 = "t3.small" 
+      asg_desired_capacity          = 2 
+      root_volume_size              = 300 
+      additional_ebs_volumes = [{
+        block_device_name = "/dev/sda1",
+        volume_size       = 300
+      }]
+
+      tags = [
+        {
+          key                 = "OS"
+          value               = "windows"
+          propagate_at_launch = true
+        },
+        {
+          key                 = "k8s.io/cluster-autoscaler/${local.name}"
+          value               = "owned"
+          propagate_at_launch = true
+        },
+        {
+          key                 = "k8s.io/cluster-autoscaler/enabled"
+          value               = true
+          propagate_at_launch = true
+        }
+      ]
+    },
+    // default Linux worker group launch template for big workloads
```

Comment:
Please add one node group or linux worker, for example:

```
  node_groups = {
    example = {
      desired_capacity = 1
      max_capacity     = 10
      min_capacity     = 1
      instance_types = ["t3.large"]
    }
}
```

as only windows cluster will be not working in example